spec/rspec/core/formatters/exception_presenter_spec.rb

```diff
@@ -230,6 +244,31 @@ module RSpec::Core
         EOS
       end
 
+      it 'will work when cause is incorrectly overridden', :if => RSpec::Support::RubyFeatures.supports_exception_cause? do
+        exception_klass = Class.new(Exception) do
+          def cause
+            "An incorrect implementation"
+          end
+        end
+        incorrect_cause_exception = exception_klass.new
```

Comment:
Can this be just `FakeException.new("Boom", [], "An incorrect implementation")`?

Reply:
Good point, this predated the refactoring exceptions to make it work :joy:

Reply:
Fixed.